task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,10 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num%15==0) return 'FizzBuzz'; 
+    if (num%5==0) return 'Buzz';
+    if (num%3==0) return 'Fizz';
+    return num; 
```

Comment:
operator `===` is always preferable 
I noticed you don't like spaces, please follow the js code style i.e. if (num % 15 === 0)